Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Isotropic_remeshing/remesh_impl.h

```diff
@@ -1528,6 +1528,13 @@ namespace internal {
           set_status(h, PATCH_BORDER);
           has_border_ = true;
         }
+        if (status(h) == PATCH
```

Comment:
you could only update the first if with
` if (status(h)==PATCH && (status(opposite(h, mesh_))!=PATCH || get_patch_id(face(h, mesh_)) != get_patch_id(face(opposite(h, mesh_), mesh_)))`

Reply:
right, thank you!